test/rust_src/src/windows-tests.el

```diff
@@ -38,6 +38,10 @@
   (should (eq (window-old-point) 1))
   (should (eq (window-old-point (selected-window)) 1)))
 
```

Comment:
Awesome. Tests rock.

Why 9? Can we use a variable or at a minimum have a comment documenting the choice

Reply:
That indeed does work, and is at least a little less magical than hard-coding a 9. It looks like the relationship between new-total and total-height is a bit more complicated than this call would imply, but that's probably well out of the scope of this MR.

Reply:
Agreed.